web/app/views.py

```diff
@@ -21,11 +21,14 @@
 from lib.file_storage import save_file_obj
 from app.tasks import preprocess_timelapse
 
+# def index(request):
+#     if request.user.is_authenticated and not request.user.consented_at:
+#         return redirect('/consent/')
+#     else:
+#         return redirect('/printers/')
+
 def index(request):
-    if request.user.is_authenticated and not request.user.consented_at:
-        return redirect('/consent/')
-    else:
-        return redirect('/printers/')
+    return render(request, 'frontend/index.html')
```

Comment:
this broke the app. please revert

Reply:
reverted. I changed it to check merging Vue front to app.